includes/Core/Site_Health/Tag_Placement.php

```diff
@@ -197,12 +190,18 @@ protected function tags_placement_test() {
 	 *
 	 * @return array Filtered active modules instances.
 	 */
-	protected function get_active_modules() {
+	protected function get_active_modules_with_tags() {
 		$active_modules = $this->modules->get_active_modules();
 
 		$active_modules = array_filter(
 			$active_modules,
 			function( $module ) {
+				// @TODO remove the check when Analytics module is removed.
+				// Temorary added to exclude analytics module in favor of analytics 4.
+				if ( 'analytics' === $module->slug ) {
+					return false;
+				}
+
 				return $module instanceof Module_With_Tag;
```

Comment:
It's possible we could simplify the above special case by removing this interface from `analytics` since it isn't being output anymore anyways. Just an idea, but let's keep it simple for now.

Reply:
It is going to be deleted soon in last SAM task anyway, agree, let's keep it till then for simplicity